drivers/wifi/nrf_wifi/src/wifi_mgmt_scan.c

```diff
@@ -278,6 +278,34 @@ static inline enum wifi_mfp_options drv_to_wifi_mgmt_mfp(unsigned char mfp_flag)
 
 	return WIFI_MFP_UNKNOWN;
 }
+
+static inline enum wifi_cipher_suite_b drv_to_wifi_mgmt_suiteb(int drv_security_type)
+{
+	switch (drv_security_type) {
+	case NRF_WIFI_OPEN:
+	case NRF_WIFI_WEP:
+	case NRF_WIFI_WPA:
+	case NRF_WIFI_WPA2:
+	case NRF_WIFI_WPA2_256:
+	case NRF_WIFI_WPA3:
+	case NRF_WIFI_WAPI:
+	case NRF_WIFI_FT_EAP_SHA384:
+	case NRF_WIFI_FT_PSK_SHA384:
+	case NRF_WIFI_PSK_SHA384:
+		return WIFI_CIPHER_SUITE_B_NA;
+	case NRF_WIFI_EAP:
+	case NRF_WIFI_EAP_TLS_SHA256:
+	case NRF_WIFI_FT_EAP:
+		return WIFI_CIPHER_SUITE_B_NONE;
```

Comment:
How to distinguish `WIFI_CIPHER_SUITE_B_NA` and `WIFI_CIPHER_SUITE_B_NONE`?
Why `NRF_WIFI_FT_EAP_SHA384` belongs to `WIFI_CIPHER_SUITE_B_NA`?